understanding/20/sign-language-prerecorded.html

```diff
@@ -11,11 +11,12 @@ <h1>Understanding Sign Language (Prerecorded)</h1>
    <section id="brief">
       <h2>In brief</h2>
       <dl>
-         <dt>Objective</dt><dd>Videos can be accompanied by sign language</dd>
-         <dt>Author task</dt><dd>Provide sign language interpretation for audio content in existing videos</dd>
+         <dt>Goal</dt><dd>Videos can be accompanied by sign language.</dd>
+         <dt>What to do</dt><dd>Provide sign language interpretation for audio content in existing videos.</dd>
+         <dt>Why it's important</dt><dd>People who are deaf or hard of hearing have more ways to understand multimedia content.</dd> 
```

Comment:
```suggestion
         <dt>Why it's important</dt><dd>People who are deaf or hard of hearing can understand multimedia in their preferred language.</dd> 
```

Reply:
People who are hard of hearing may not understand sign language at all. And there are many different sign languages. The suggested change "...can understand ... in their preferred language" seems over-promise. I prefer "have more ways."